libs/git/repository.go

```diff
@@ -23,8 +23,18 @@ type Repository struct {
 	// directory where we process .gitignore files.
 	real bool
 
-	// root is the absolute path to the repository root.
-	root vfs.Path
+	// checkoutDir is the path to the root of the repository checkout.
+	// This can be either the main repository checkout or a worktree.
+	checkoutDir vfs.Path
```

Comment:
Can you add more details/examples for the two definitions here (worktree vs vanilla)? Maybe just links are enough?

An uninitiated reader would not know what a checkout of a worktree is.

Reply:
Updated the comment and added a link to the worktree docs.